addons/calendar/static/src/views/calendar_form/calendar_form_controller.js

```diff
@@ -38,4 +39,24 @@ export class CalendarFormController extends FormController {
         }
         return super.beforeExecuteActionButton(...arguments);
     }
+
+    /**
+     * Custom delete function for calendar events, which can call the unlink action or not.
+     * When if (rootValues.attendees_count == 1 && rootValues.user_id[0] !== rootValues.partner_ids._currentIds[0], it performs the default delete.
```

Comment:
You should describe this in words instead of the code itself, because it is **not** trivial for other devs
:warning:

Reply:
@geflx Done. :+1: